src/SFA.DAS.AssessorService.Data/EMailTemplateQueryRepository.cs

```diff
@@ -1,24 +1,30 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿using Dapper;
 using System.Threading.Tasks;
-using Microsoft.EntityFrameworkCore;
 using SFA.DAS.AssessorService.Application.Interfaces;
-using SFA.DAS.AssessorService.Domain.Entities;
+using SFA.DAS.AssessorService.Domain.DTOs;
+using System.Linq;
 
 namespace SFA.DAS.AssessorService.Data
 {
-    public class EMailTemplateQueryRepository : IEMailTemplateQueryRepository
+    public class EMailTemplateQueryRepository : Repository, IEMailTemplateQueryRepository
     {
-        private readonly AssessorDbContext _assessorDbContext;
-
-        public EMailTemplateQueryRepository(AssessorDbContext assessorDbContext)
+        public EMailTemplateQueryRepository(IUnitOfWork unitOfWork) : base(unitOfWork)
         {
-            _assessorDbContext = assessorDbContext;
         }
 
-        public async Task<EMailTemplate> GetEmailTemplate(string templateName)
+        public async Task<EmailTemplateSummary> GetEmailTemplate(string templateName)
         {
-            return await _assessorDbContext.EMailTemplates.FirstOrDefaultAsync(q => q.TemplateName == templateName);
+            var sql = @"SELECT ET.Id, ET.TemplateName, ET.TemplateId, ETR.Recipients, ET.RecipientTemplate
+                    FROM [dbo].[EMailTemplates] ET join [dbo].[EmailTemplatesRecipients] ETR
+                    ON ET.Id = ETR.EmailTemplateId
+                    WHERE ET.TemplateName = @templateName";
+
+            var results = await _unitOfWork.Connection.QueryAsync<EmailTemplateSummary>(
+                sql,
+                param: new { templateName },
+                transaction: _unitOfWork.Transaction);
+
+            return results.FirstOrDefault();
```

Comment:
The underlying table structure may include more than one recipient row for an email template - there is nothing to stop that, so taking the first is a potential bug waiting to happen - suggest you add a unique key to the table to prevent that ever happening

Reply:
done